content/6_Advanced/Extend_Euclid.mdx

```diff
@@ -112,6 +174,19 @@ ll euclid(ll a, ll b) {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+
+def euclid(a: int, b: int) -> int:
+    """Recursive Euclidean GCD."""
+    if not b:
+        return a
+    return euclid(b, a%b)
```

Comment:
```suggestion
    return euclid(b, a % b)
```

Reply:
if sans or envy told you to do otherwise, ignore this

Reply:
See: https://peps.python.org/pep-0008/#other-recommendations

Reply:
https://usaco.guide/general/adding-solution?lang=py#code-conventions